services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from django.shortcuts import get_object_or_404
+
+import init_django_orm  # noqa: F401
+from db.models import User
```

Comment:
Avoid importing `User` directly. Use `get_user_model()` to ensure consistency with Django's user model configuration.